pos_receipt_replace_user_by_trigram/models/res_users.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    trigram = fields.Char(string="Trigram", compute="_compute_trigram")
```

Comment:
can you please show the trigram in the res.users form as readonly under First Name field?